packages/create-platformatic-db/src/create-package-json.mjs

```diff
@@ -0,0 +1,33 @@
+import pupa from 'pupa'
+import { isFileAccessible } from './utils.mjs'
+import { writeFileSync } from 'node:fs'
+import { join } from 'node:path'
+
+const packageJsonTemplate = `
+{
+  "name": "platformatic-db-api-example",
```

Comment:
Maybe we should about it in the prompt or use the folder name.

Reply:
Or not setting the name at all (which is legit). What do you think @simonplend ?

Reply:
I'm ok with it.

Reply:
Using the folder name seems reasonable.